run_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
How is this file run? I can't see anything that is executing it?

Reply:
is a drop-in template. https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-test-script

Internally conda generates a `conda_test_runner.sh` that is used to run it.